Code/RDGeneral/RDLog.h

```diff
@@ -90,18 +92,14 @@ RDKIT_RDGENERAL_EXPORT std::ostream &toStream(std::ostream &);
   RDLog::toStream((__arg__->teestream) ? *(__arg__->teestream)  \
                                        : *(__arg__->dp_dest))
 
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdAppLog;
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdDebugLog;
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdInfoLog;
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdErrorLog;
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdWarningLog;
-RDKIT_RDGENERAL_EXPORT extern std::shared_ptr<boost::logging::rdLogger>
-    rdStatusLog;
+typedef std::shared_ptr<boost::logging::rdLogger> RDLogger;
```

Comment:
I think we ought to switch to using `using` instead of `typedef` when we write new code
I will at some point do a `clang-tidy` pass across the code base to take care of the legacy, but we might as well not make that legacy larger. ;-)

Reply:
@greglandrum I'm not quite sure what you mean by "linked to the issue" do you want me to put in an rdkit issue and link to that?

Reply:
@bp-kelley : isn't this connected to #3218?